pkg/healthcheck/healthcheck.go

```diff
@@ -729,6 +735,14 @@ func (hc *HealthChecker) allCategories() []category {
 						return hc.validateServiceProfiles()
 					},
 				},
+				{
+					description: "tap api service is running",
+					hintAnchor:  "#",
```

Comment:
This will require a corresponding change in the website repo to add the hint help.

```
‼ tap api service is running
    apiservices.apiregistration.k8s.io "v1alpha1.tap.linkerd.io" not found
    see https://linkerd.io/checks/## for hints
```

Reply:
I don't know what needs to be in the hint help. If you can provide me with the content I'll add a PR to that repo. For now I'll add a hint anchor